### ISSUE TITLE
Settings of the hyperparameters

### ISSUE THREAD

**Issue by lkastar** [Aitical/Task-agnostic_Model_Contrastive_Learning_Image_Restoration#4]
would you plz tell me the settings of the hyperparameters `lambda`、`update_setps` and the `updating weight` of EMA? 

thx for ur contribution!

**Comment by Aitical** [comment]
Thank you for your interest in our work. Apologies for the delayed response.

First, regarding the three hyperparameters you mentioned - lambda, update step, and EMA weight - we have conducted ablation studies on some tasks in our paper, which you may refer to.

Additionally, the MCLIR training process is task-agnostic. Apart from the ablation results shown for some tasks, when training on different tasks, some parameters need to be modified. This is mainly because different tasks have varying convergence behaviors. Here, we share some empirical choices we made during our experiments:

1. Loss weight lambda: I generally control the negative sample loss value to be within the range of 0.01 to 0.1 times the target reconstruction loss.
2. Update step: This is a robust parameter, and as we pointed out in the paper, MCLIR is more stable and performs better when extended to multiple negative models. Please refer to the ablation part of the paper.
3. EMA update weight: Similarly, we used the same weight for all experiments, which you can find in the ablation part of the paper.
